config/prod/tthyer-arvados.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: tthyer-arvados-trial
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad/arvados-trial"
```

Comment:
Does / work? @zaro0508

Reply:
This works. This is the way @zaro0508 recommended that Rob indicate a sub-project.

Reply:
Yes, this is fine to do.